### PR TITLE
Fixes the exosuit autolathe being unable to pick up lead, aluminium or plastic

### PR DESCRIPTION
**Pull request by Snowy1237**
Just another simple bugfix.

### CODE REVIEW

**Comment by Snowy1237** [comment]
!review

**Comment by Snowy1237** [comment]
!bugfix

**Review by alsoandanswer — APPROVED** [review]

File: code/modules/heavy_vehicle/equipment/utility.dm [hunk 1]
```diff
@@ -621,7 +621,7 @@
 
 /obj/item/mecha_equipment/autolathe/afterattack(atom/target, mob/living/user, inrange, params)
 	. = ..()
-	if(istype(target, /obj/item/stack/material/steel) || istype(target, /obj/item/stack/material/glass))
+	if(istype(target, /obj/item/stack/material/steel) || istype(target, /obj/item/stack/material/glass) || istype(target, /obj/item/stack/material/aluminium) || istype(target, /obj/item/stack/material/lead) || istype(target, /obj/item/stack/material/plastic))
```
Comment (FluffyGhoster) [inline code/modules/heavy_vehicle/equipment/utility.dm:624]: Use `is_type_in_list` instead of 30 istype OR'd

**Review by FluffyGhoster — CHANGES_REQUESTED** [review]

File: code/modules/heavy_vehicle/equipment/utility.dm (hunk 1 above)
Comment (Snowy1237) [reply code/modules/heavy_vehicle/equipment/utility.dm]: should be better now, thanks for telling me

**Review by NonQueueingMatt — APPROVED** [review]

**Pull request merged**